bin/lerna.js

```diff
@@ -22,6 +22,7 @@ var cli = meow([
   "  --canary, -c         Publish packages after every successful merge using the sha as part of the tag",
   "  --skip-git           Skip commiting, tagging, and pushing git changes (only affects publish)",
   "  --npm-tag [tagname]  Publish packages with the specified npm dist-tag",
+  "  --restrict-to [package glob] Restricts a command to run only packages matching the given glob (Works only in combination with the 'run' command).",
```

Comment:
I'm not sure about the name `--restrict-to`, any other ideas?

---

Also (separate from this PR), we really should fine a better way of documenting flags within the CLI help.